qt/applications/workbench/workbench/plugins/memorywidget/memoryview.py

```diff
@@ -0,0 +1,58 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2017 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+#  This file is part of the mantid workbench.
+#
+#
+from qtpy.QtWidgets import QWidget, QProgressBar
+from qtpy.QtCore import Qt
+
+NORMAL_STYLE = """
+QProgressBar::chunk {
+    background-color: lightgreen;
+}
+"""
+
+CRITICAL_STYLE = """
+QProgressBar::chunk {
+    background-color: red;
+}
+"""
+
+
+def from_normal_to_critical(critical, current_value, new_value) -> bool:
+    return current_value < critical <= new_value
+
+
+def from_critical_to_normal(critical, current_value, new_value) -> bool:
+    return current_value >= critical > new_value
+
+
+class MemoryView(QWidget):
+    def __init__(self, parent):
+        super(MemoryView, self).__init__(parent)
+
+        self.critical = 90
```

Comment:
Is this `90` the same meaning as in the presenter? Can we share this by passing it in to view in the `__init__` method?